src/patterns/utils.rs

```diff
@@ -121,11 +118,7 @@ pub fn require_node_property(node: &Node, property: &str) -> Result<String, Stri
 }
 
 /// generates a specific error message if `property.is_some()`
-pub fn check_found_property_already(
-  property: &Option<String>,
-  node_name: &str,
-  parent_name: &str,
-) -> Result<(), String> {
+pub fn check_found_property_already(property: &Option<String>, node_name: &str, parent_name: &str) -> Result<(), String> {
```

Comment:
Similar for these function signatures, a bit torn which way is the more convenient way to read them in the long run